src/Handler/RetryHandler.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Mollie       https://www.mollie.nl
+ *
+ * @author      Mollie B.V. <info@mollie.nl>
+ * @copyright   Mollie B.V.
+ * @license     https://github.com/mollie/PrestaShop/blob/master/LICENSE.md
+ *
+ * @see        https://github.com/mollie/PrestaShop
+ * @codingStandardsIgnoreStart
+ */
+
+namespace Mollie\Handler;
+
+use Mollie\Exception\RetryOverException;
+
+class RetryHandler
+{
+    const DEFAULT_MAX_RETRY = 3;
+    const DEFAULT_WAIT_TIME = 1;
+    const DEFAULT_ACCEPTED_EXCEPTION = 'RuntimeException';
+
+    /**
+     * @var callable
+     */
+    protected $_proc;
```

Comment:
why is $_ used? simply could use different name or smth, _ was used to specify private variables in early PHP